zephyr/module.yml

```diff
@@ -194,7 +194,7 @@ blobs:
     description: "Modified bootrom to boot Blackhole"
     doc-url: https://tenstorrent.com
   - path: fw_pack-wormhole.tar.gz
-    sha256: 2e321cbdea7bbafcc41e2ce39d5ce4d6a38a7669dc18c3aad3543fb274d2f45b
+    sha256: b27e10380c7769ceab3c924da3df75620b80269d58a7b82dd6ed1df30dd9d345
     type: img
     version: '80.18.0.0'
```

Comment:
Maybe `18.4.0-rc1` for the version? I was planning on tagging RC1 tonight.

Reply:
Yeah, I'll update it